tests/mixed_int8/README.md

```diff
@@ -1,37 +1,122 @@
 # Testing mixed int8 quantization
 
+![HFxbitsandbytes.png](https://s3.amazonaws.com/moonup/production/uploads/1660567705337-62441d1d9fdefb55a0b7d12c.png)
+
+Hi there, this is a recipe on how to effectively debug `bitsandbytes` integration on HuggingFace `transformers`.
+
+## Library requirements
+
++ `transformers>=4.22.0`
++ `accelerate>=0.12.0` 
++ `0.31.8>=bitsandbytes>=0.31.5`.
 ## Hardware requirements
 
-I am using a setup of 2 GPUs that are NVIDIA-Tesla T4 15GB
+I am using a setup of 2 GPUs that are NVIDIA-Tesla T4 15GB - `younes-testing-multi-gpu` on GCP. To run successfully `bitsandbytes` you would need a 8-bit core tensor supported GPU. Note that Ampere architectures should be supported. Here is an exhaustive list of the supported GPU types at the time of this writing:
+
+- RTX 20s & RTX 30s
+- A40-A100
+- T4 + (e.g. Google Colab GPUs) 
 
 ## Virutal envs
 
 ```conda create --name int8-testing python==3.8```
-```git clone https://github.com/younesbelkada/transformers.git && git checkout integration-8bit```
-```pip install -e ".[dev]"```
-```pip install -i https://test.pypi.org/simple/ bitsandbytes```
-```pip install git+https://github.com/huggingface/accelerate.git@e0212893ea6098cc0a7a3c7a6eb286a9104214c1```
-
+```pip install bitsandbytes>=0.31.5```
+```pip install accelerate>=0.12.0```
+```pip install git+https://github.com/huggingface/transformers.git```
+if `transformers>=4.22.0` is not released yet, otherwise
+```pip install transformers```
 
 ## Trobleshooting
 
+A list of common errors:
+
+### Torch does not correctly do the operations on GPU
+
+First check that:
+
+```py
+import torch
+
+vec = torch.randn(1, 2, 3).to(0)
+```
+
+Works without any error. If not install torch using `conda` like:
+
 ```conda create --name int8-testing python==3.8```
-```pip install -i https://test.pypi.org/simple/ bitsandbytes```
+```pip install bitsandbytes```
 ```conda install pytorch torchvision torchaudio -c pytorch```
-```git clone https://github.com/younesbelkada/transformers.git && git checkout integration-8bit```
-```pip install -e ".[dev]"```
-```pip install git+https://github.com/huggingface/accelerate.git@b52b793ea8bac108ba61192eead3cf11ca02433d```
+```pip install git+https://github.com/huggingface/transformers.git```
+```pip install accelerate```
+
+And the snippet above should work
+
+### ` `bitsandbytes` operations are not supported under CPU!`
```

Comment:
The usage of **`** here makes the rendered version (GitHub page) looks strange.

see https://github.com/younesbelkada/transformers/blob/bnb-fix-small-details/tests/mixed_int8/README.md

Reply:
Thanks for spotting this!
Should be fixed in c8d628195bd9fca4ae25031205d220aa2248ab70